agent/tool-scripts/postprocess/jmap-postprocess

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
```

Comment:
Looks like the postprocess script is just adding a header to the stdout of the tool script. Since the output of the tool is in csv format, can you modify the postprocess script to use the existing visualization capability in pbench to visualize the data?

Reply:
can you elaborate a little more ?

Reply:
@mrsiano if you look at iostat tool for example, it has a postprocess script which outputs html files using the data from the .txt files - https://github.com/distributed-system-analysis/pbench/blob/master/agent/tool-scripts/postprocess/iostat-postprocess. Can you modify the tool to do the same?

Reply:
we'll have a follow up PR.